src/general/global.opy

```diff
@@ -4,6 +4,7 @@
 subroutine CheckStatus
 
 
+#this is useless right?
```

Comment:
Why? It's used in several places. I think for Winston and for Mei only at the moment

Reply:
It is a noop, it just does a bunch of ifs then "return".

Reply:
I thought the same, but didn't make it so I assumed it worked, but I will do some testing to see if really does work.

Reply:
Update; it doesn't work